### PR TITLE
refactor, chore(egen): Removes boilerplate, heading fixes, and other corrections from template

### PR DESCRIPTION
**Pull request by nileshspringml**
**REQUIRED:** 

- Updated template of notebook based on new template
- Tested code end to end and its working fine

<br>
Notebook template is updated and tested notebook end to end
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/custom/sdk-custom-image-classification-batch.ipynb [hunk 1]
```diff
@@ -32,25 +33,27 @@
...
+        "      <img src=\"https://cloud.google.com/ml-engine/images/colab-logo-32px.png\" alt=\"Google Colaboratory logo\"><br> Open in Colab\n",
         "    </a>\n",
         "  </td>\n",
-        "  <td>\n",
-        "    <a href=\"https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/custom/sdk-custom-image-classification-batch.ipynb\">\n",
-        "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
-        "      View on GitHub\n",
+        "  <td style=\"text-align: center\">\n",
+        "    <a href=\"https://console.cloud.google.com/vertex-ai/colab/import/https:%2F%2Fraw.githubusercontent.com%2FGoogleCloudPlatform%2Fvertex-ai-samples%2Fmain%2Fnotebooks%2Fofficial%2Fcustom%2Fsdk-custom-image-classification-batch.ipynb\">\n",
+        "      <img width=\"32px\" src=\"https://lh3.googleusercontent.com/JmcxdQi-qOpctIvWKgPtrzZdJJK-J3sWE1RsfjZNwshCFgE_9fULcNpuXYTilIR2hjwN\" alt=\"Google Cloud Colab Enterprise logo\"><br> Open in Colab Enterprise\n",
```
Comment (kittyabs) [inline notebooks/official/custom/sdk-custom-image-classification-batch.ipynb:43]: change logo (colab enterprise) URL to:

- https://cloud.google.com/ml-engine/images/colab-enterprise-logo-32px.png

File: notebooks/official/custom/sdk-custom-image-classification-batch.ipynb [hunk 2]
```diff
@@ -78,7 +81,7 @@
         "In this tutorial, you learn to use Vertex AI Training to create a custom trained model and use Vertex AI Batch Prediction to do a batch prediction on the trained model.\n",
         "\n",
         "\n",
-        "create a custom-trained model from a Python script in a Docker container using the Vertex AI SDK for Python, and then do a prediction on the deployed model by sending data. Alternatively, you can create custom-trained models using `gcloud` command-line tool, or online using the Cloud Console.\n",
+        "Create a custom-trained model from a Python script in a Docker container using the Vertex AI SDK for Python, and then do a prediction on the deployed model by sending data. Alternatively, you can create custom-trained models using `gcloud` command-line tool, or online using the Cloud Console.\n",
```
Comment (kittyabs) [inline notebooks/official/custom/sdk-custom-image-classification-batch.ipynb:84]: Vertex AI SDK for Python

File: notebooks/official/custom/sdk-custom-image-classification-batch.ipynb [hunk 3]
```diff
@@ -128,182 +131,124 @@
...
-        "id": "1fd00fa70a2a"
-      },
-      "outputs": [],
-      "source": [
-        "# Install the packages\n",
-        "! pip3 install --upgrade google-cloud-aiplatform \\\n",
-        "                        google-cloud-storage \\\n",
-        "                        pillow  \\\n",
-        "                        numpy    "
+        "## Getting Started"
```
Comment (kittyabs) [inline notebooks/official/custom/sdk-custom-image-classification-batch.ipynb:137]: Get started 

- no gerunds in headers
- lower case for "started"

File: notebooks/official/custom/sdk-custom-image-classification-batch.ipynb [hunk 4]
```diff
@@ -128,182 +131,124 @@
...
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "d98bc9fdd80d"
+        "id": "aae9ca040eab"
       },
       "source": [
-        "### Colab only: Uncomment the following cell to restart the kernel."
+        "### Install Vertex AI SDK and other required packages\n"
```
Comment (kittyabs) [inline notebooks/official/custom/sdk-custom-image-classification-batch.ipynb:146]: Vertex AI SDK for Python

File: notebooks/official/custom/sdk-custom-image-classification-batch.ipynb [hunk 5]
```diff
@@ -128,182 +131,124 @@
...
       "cell_type": "markdown",
       "metadata": {
-        "id": "af349043f23b"
+        "id": "54c5ef8a8f43"
       },
       "source": [
-        "**1. Vertex AI Workbench**\n",
-        "* Do nothing as you are already authenticated."
+        "<div class=\"alert alert-block alert-warning\">\n",
+        "<b>⚠️ The kernel is going to restart. Please wait until it is finished before continuing to the next step. ⚠️</b>\n",
```
Comment (kittyabs) [inline notebooks/official/custom/sdk-custom-image-classification-batch.ipynb:199]: Remove "Please" and contract "it is":

- Wait until it's finished before...

File: notebooks/official/custom/sdk-custom-image-classification-batch.ipynb (hunk 1 above)
Comment (nileshspringml) [reply notebooks/official/custom/sdk-custom-image-classification-batch.ipynb]: Logo is updated

File: notebooks/official/custom/sdk-custom-image-classification-batch.ipynb (hunk 3 above)
Comment (nileshspringml) [reply notebooks/official/custom/sdk-custom-image-classification-batch.ipynb]: Did required changes

File: notebooks/official/custom/sdk-custom-image-classification-batch.ipynb (hunk 4 above)
Comment (nileshspringml) [reply notebooks/official/custom/sdk-custom-image-classification-batch.ipynb]: Did required changes

File: notebooks/official/custom/sdk-custom-image-classification-batch.ipynb (hunk 5 above)
Comment (nileshspringml) [reply notebooks/official/custom/sdk-custom-image-classification-batch.ipynb]: Did required changes

File: notebooks/official/custom/sdk-custom-image-classification-batch.ipynb (hunk 2 above)
Comment (nileshspringml) [reply notebooks/official/custom/sdk-custom-image-classification-batch.ipynb:84]: @kittyabs could you please elaborate this comment. Which particular change we need to make?

**Comment by katiemn** [comment]
Is this draft pull request ready for review?

**Comment by nileshspringml** [comment]
> Is this draft pull request ready for review?

Yes, ready for PR review

**Review by katiemn — APPROVED** [review]

**Pull request merged**